src/_includes/core-nodes/mcp-prompt-use-case.md

```diff
@@ -0,0 +1,181 @@
+
+### Prompt Template
+
+`string` — *Required*
+
+{% raw %}
+Define your prompt text using **double curly braces `{{ }}`** for variables.
+{% endraw %}
+
+### Arguments
+
+`JSON` — *Required*
+JSON schema defining the template variables that users can customize. This follows the same **JSON Schema** format used in MCP Tool input schemas.
+
+## Prompt Template Examples
+
+### Basic Example
+
+```
+Hello {{ name }}! Welcome to {{ location }}.
+```
+
+Variables: `name`, `location`
+
+### Complex Example
+
+```
+You are a holiday planning agent.
+You should provide information about {{ location }}.
+You should also provide rough budget ideas for visiting this place in {{ time_of_year }} for {{ duration }} days.
+Please break down rough ideas for hotels and local tourist hot spots to visit.
+```
+
+Variables: `location`, `time_of_year`, `duration`
+
+### Multi-Section Example
+
+```
+# Code Review Request
+
+## File: {{ filename }}
+## Language: {{ language }}
+
+Please review the following code:
+
+{{ code }}
+
+Focus on:
+- {{ focus_area_1 }}
+- {{ focus_area_2 }}
+- {{ focus_area_3 }}
+
+Provide feedback on code quality, potential bugs, and suggestions for improvement.
+```
+
+Variables: `filename`, `language`, `code`, `focus_area_1`, `focus_area_2`, `focus_area_3`
+
+## Argument Schema Examples
+
+### Basic Schema
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Your name",
+      "minLength": 1
+    },
+    "location": {
+      "type": "string",
+      "description": "The place you're visiting",
+      "minLength": 1
+    }
+  },
+  "required": ["name", "location"]
+}
+```
+
+### Extended Schema
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "location": {
+      "type": "string",
+      "description": "A country, city or town somewhere in the world",
+      "minLength": 1
+    },
+    "time_of_year": {
+      "type": "string",
+      "description": "A specific date, season or month giving context for the travel period",
+      "minLength": 1
+    },
+    "duration": {
+      "type": "number",
+      "default": 7,
+      "description": "The number of days for the trip"
+    }
+  },
+  "required": ["location", "time_of_year"]
+}
+```
+
+### Schema with Enums and Defaults
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "report_type": {
+      "type": "string",
+      "description": "Type of report to generate",
+      "enum": ["daily", "weekly", "monthly", "quarterly"],
+      "default": "weekly"
+    },
+    "include_charts": {
+      "type": "boolean",
+      "description": "Include visual charts in the report",
+      "default": true
+    },
+    "detail_level": {
+      "type": "string",
+      "description": "Level of detail for the report",
+      "enum": ["summary", "detailed", "comprehensive"],
+      "default": "detailed"
+    }
+  },
+  "required": ["report_type"]
+}
+```
+
+## Example Flow
+
+{% renderFlow %}
```

Comment:
for some reason this does not render

<img width="1130" height="600" alt="Image" src="https://github.com/user-attachments/assets/b8eba7b8-9c39-4f00-bba8-f75d6168c839" />